src/LrtSquare.sol

```diff
@@ -413,6 +501,23 @@ contract LrtSquare is
         return IERC20Metadata(erc20).decimals();
     }
 
+    function _checkPercentagesInVault() internal view {
```

Comment:
lets make it more descriptive. 

- `_checkPercentagesInVault` -> `_verifyPositionLimits`
- `maxPercentageInVault` -> `positionLimitPercentage`

Reply:
Done